### PR TITLE
docs: update rainbond docs

### PR DESCRIPTION
**Pull request by zzzhangqi**
## Purpose of the pull request

Updated the documentation for installing Dolphinscheduler with Rainbond.

### CODE REVIEW

**Comment by zzzhangqi** [comment]
This change in documentation applies to all Dolphinscheduler versions.

Rainbond community keeps an eye on Dolphinscheduler updates and continues to update to the latest version.

🤗🤗

File: docs/docs/zh/guide/integration/rainbond.md [hunk 1]
```diff
@@ -8,28 +8,26 @@
...
-* 点击 DolphinScheduler 右侧的 `安装` 进入安装页面，填写对应的信息，点击确定即可开始安装，自动跳转至应用视图。
+2. 点击 DolphinScheduler 右侧的**安装**进入应用安装页面，填写对应的信息，点击确定即可开始安装，自动跳转至应用视图。
 
 | 选择项  |                      说明                      |
 |------|----------------------------------------------|
 | 团队名称 | 用户自建的工作空间，以命名空间隔离                            |
 | 集群名称 | 选择 DolphinScheduler 被部署到哪一个 K8s 集群           |
 | 选择应用 | 选择 DolphinScheduler 被部署到哪一个应用，应用中包含有若干有关联的组件 |
-| 应用版本 | 选择 DolphinScheduler 的版本，目前可选版本为 3.0.0-beta2  |
+| 应用版本 | 选择 DolphinScheduler·的版本                      |
```
Comment (EricGao888) [inline docs/docs/zh/guide/integration/rainbond.md]:
```suggestion
| 应用版本 | 选择 DolphinScheduler 的版本                      |
```

File: docs/docs/en/guide/integration/rainbond.md [hunk 2]
```diff
@@ -23,13 +23,11 @@ This section describes the one-click deployment of high availability DolphinSche
...
 
-* Wait a few minutes, the installation completes and `DolphinScheduler` will be running.
+3. Wait a few minutes, the installation completes and `DolphinScheduler` will be running.
 
 ![](../../../../img/rainbond/topology-dolphinscheduler.png)
 
-* Access DolphinScheduler-API components and the default user credentials are `admin` / `dolphinscheduler123`.
-
-![](../../../../img/rainbond/homepage-dolphinscheduler.png)
+4. Click the in app access button to access the DolphinScheduler-API components through the domain name provided by Rainbond by default, default user password is **admin/dolphinscheduler123**.
```
Comment (EricGao888) [inline docs/docs/en/guide/integration/rainbond.md]:
```suggestion
4. Click the `access` button in the application to access the DolphinScheduler-API components through the domain name provided by Rainbond by default. Default user password is **admin/dolphinscheduler123**.
```

**Comment by zzzhangqi** [comment]
@EricGao888 resolved

**Review by EricGao888 — APPROVED** [review]

**Comment by EricGao888** [comment]
Thanks @zzzhangqi

**Pull request merged**